examples/hello_world/client/node_js/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Is it worth moving this entry to the top-level gitignore? I guess it will have to be repeated for any additional NodeJS module anywhere in the repo anyways, if we had more? Up to you.

Reply:
Happy to add it to the global .gitignore, seems like a good idea. :)